packages/office-ui-fabric-react/src/components/GroupedList/index.ts

```diff
@@ -10,3 +10,5 @@ export { IGroupShowAllStyleProps, IGroupShowAllStyles } from './GroupShowAll.typ
 export { GroupSpacer } from './GroupSpacer';
 export * from './GroupSpacer.types';
 export * from './GroupedListSection';
+
+export { GroupedListV2_unstable } from './GroupedListV2';
```

Comment:
We discussed elsewhere but since v8 isn't exporting these things I'll leave it off here as well. I'll make a follow up PR to export these from v8 and then cherry pick that into v7.

#26041